src/Core/DevOps/System/Command/SystemGenerateJwtSecretCommand.php

```diff
@@ -28,7 +28,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
         if (!\extension_loaded('openssl')) {
             $io->error('extension openssl is required');
 
-            return 1;
+            self::FAILURE;
```

Comment:
you missed the return for all failure cases 🙂

Reply:
Whoops! Fixed